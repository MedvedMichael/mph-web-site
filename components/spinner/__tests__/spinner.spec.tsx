import { shallow } from 'enzyme'
import Spinner from '../spinner'
import toJson from 'enzyme-to-json'

test('should render corr', () => {
    const res = shallow(<Spinner/>)
    expect(toJson(res)).toMatchSnapshot()
})