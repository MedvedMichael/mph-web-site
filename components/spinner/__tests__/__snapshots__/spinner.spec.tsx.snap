// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render corr 1`] = `
<styled.div>
  <styled.div>
    <div />
    <div />
  </styled.div>
</styled.div>
`;
